packages/docbook_xml50.rb

```diff
@@ -189,20 +206,21 @@ def self.install
       done
     ADDFILES_HEREDOC
     IO.write('add_files.sh', @ADDFILES_SH, perm: 0o755)
-    system "#{CREW_PREFIX}/bin/bash -x ./add_files.sh | true"
+    system 'bash -x ./add_files.sh || true'
```

Comment:
Is calling bash necessary?  I suppose it would be necessary if `add_files.sh` is not executable.

Reply:
Original yes (although `add_files.sh` is executable, a shebang line is missing, so it need to be involved by a shell)

The missing shebang lines were added now (the file will be executed directly now)